src/OpenSage.Game/Data/InstallationLocator.cs

```diff
@@ -111,7 +112,7 @@ private static string GetRegistryValue(string keyName, string valueName)
         private IEnumerable<string> GetValidPaths(IEnumerable<string> paths)
         {
             return paths
-                .Where(p => !string.IsNullOrWhiteSpace(p))
+                .WhereNot(string.IsNullOrWhiteSpace)
```

Comment:
It's subjective, but to me this isn't as clear as the original code - probably just because I'm very used to seeing `Where` and I'm not used to seeing `WhereNot`.

Reply:
Yeah, it's definitely subjective. I prefer avoiding lambdas (which is pretty difficult in C#) when not necessary, and in my opinion `!` is easier to miss than `not`.

There are options, of course. I did consider using the name `Reject` instead, as it's used in many JS FP libraries such as Lodash and Ramda. LISPs including Clojure and Emacs LISP call it`remove`.